sopel_waifu/waifu.json

```
@@ -1487,21 +1487,70 @@
     ],
     "Touhou Project": [
         "Cirno",
-        "Marisa",
-        "Reimu Hakurei"
+        "Kirisame Marisa",
+        "Hakurei Reimu",
+        "Patchouli Knowledge",
+        "Izayoi Sakuya",
+        "Alice Margatroid",
+        "Remilia Scarlet",
+        "Hong Meiling",
+        "Kazami Yuuka",
+        "Flandre Scarlet",
+        "Letty Whiterock",
+        "Lily White",
+        "Konpaku Youmu",
+        "Saigyouji Yuyuko",
+        "Yakumo Ran",
+        "Yakumo Yukari",
+        "Maribel Hearn",
+        "Usami Renko",
+        "Ibuki Suika",
+        "Kurokoma Saki"
     ],
     "True Tears": [
-        "Hiromi Yuasa",
-        "Noe Isurugi"
+        "Yuasa Hiromi",
+        "Isurugi Noe",
+        "Andou Aiko"
     ],
     "Uchuu Kyoudai": [
         "Serika Itou"
     ],
     "Upotte!": [
```

Another one from IRC discussion before squash.

```suggestion
    "Upotte!!": [
```